### PR TITLE
Add web structure for to-do-list

### PR DESCRIPTION
**Pull request by AbdusSattar-70**
### According to the project requirement, I have made a to-do list structure.

- [ ] using the HTML Webpack Plugin.
- [ ] using Webpack
- [ ] using style, CSS, and Babel loaders

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    clean: true,
+  },
+  optimization: {
+    runtimeChunk: 'single',
+  },
+  plugins: [
+    new HtmlWebpackPlugin({
+      template: path.resolve(__dirname, './src/index.html'),
+      filename: 'toDoList.html',
+    }),
```
Comment (Omar-Muhamad) [inline webpack.config.js:25]: - [ ] When I run the dev server I get a route page including all files inside the `dist` directory (see the image below), Please remove line 24 to make the todo list HTML page load directly when running the dev server.

image

**Review by Omar-Muhamad — CHANGES_REQUESTED** [review]
# Changes Required ♻️
### Hi @AbdusSattar-70,

Nice job, you are almost there.



## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- You used the correct workflow.  ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- [ ] Please check the inline comments below.

## [Optional] suggestions:

- Nothing to mention.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: webpack.config.js (hunk 1 above)
Comment (AbdusSattar-70) [reply webpack.config.js]: Hello @Omar-Muhamad ,
Thanks for investing time to review and give suggestions.

**Review by Davidosky007 — APPROVED** [review]
Hi @AbdusSattar-70 👋 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
* No linter errors 👍 
* Well written PR description 👍 
* Good looking readme file 👍 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by AbdusSattar-70** [comment]
Hello @Davidosky007, Thanks for reviewing and approving.😍

**Pull request merged**